frontend/ui/elements/menu_activate.lua

```diff
@@ -55,7 +54,21 @@ return {
                 UIManager:show(InfoMessage:new{
                     text = _("This will take effect on next restart."),
                 })
-            end
+            end,
+            separator = true,
+        },
+        {
+            text = _("Show bottom menu"),
+            checked_func = function()
+                return G_reader_settings:readSetting("show_bottom_menu") ~= false
+            end,
+            callback = function()
+                local disabled = G_reader_settings:readSetting("show_bottom_menu") ~= false
+                G_reader_settings:saveSetting("show_bottom_menu", not disabled)
```

Comment:
Or just `G_reader_settings:flipNilOrTrue("show_bottom_menu")` (see frontend/luasettings.lua for all these helpers).
And no need for the infomessage.